src/lib/routes/admin-api/context.ts

```diff
@@ -65,6 +65,9 @@ export class ContextController extends Controller {
             middleware: [
                 openApiService.validPath({
                     tags: ['Context'],
+                    summary: 'Gets configured context fields',
```

Comment:
Would this be correct?

```suggestion
                    summary: 'Get all context fields',
```